sonar-project.properties

```diff
@@ -11,7 +11,7 @@ sonar.projectVersion=1.0
 # Path is relative to the sonar-project.properties file. Replace “\” by “/” on Windows.
 # This property is optional if sonar.modules is set.
 sonar.sources=src
```

Comment:
Just FYI, you can use wildcards in here e.g. TestResultsSchema*.js

Reply:
Thank you!